scripts/mumble-version.py

```diff
@@ -56,6 +56,7 @@
 # 1.3.0~2020-07-02~g4f336a2~snapshot.
 
 from __future__ import (unicode_literals, print_function, division)
+from optparse import OptionParser
```

Comment:
The other python scripts use argparse instead. Maybe it's worth to be consistent for things like that to increase maintainability? :eyes:

Reply:
I actually took inspiration from https://github.com/mumble-voip/mumble/blob/master/macx/scripts/osxdist.py.

The signing scripts in https://github.com/mumble-voip/mumble-releng use it as well.

Reply:
Ah okay so I guess then we are using 2 different styles already :shrug:

Reply:
I just found https://stackoverflow.com/questions/3217673/why-use-argparse-rather-than-optparse and according to that optparse got deprecated in Python 2.7
Thus imo we should try to not use it anymore :point_up:

Reply:
I tested the script with Python 3.8.7.

https://docs.python.org/3/library/optparse.html

Reply:
I said that it seems to be _deprecated_, not that it has been removed :point_up: 
And quoting from the page you linked:

> Deprecated since version 3.2: The optparse module is deprecated and will not be developed further; development will continue with the argparse module.

Given that this is a new script, I think we should not use deprecated libraries in it :)

Reply:
Ah, sorry, I missed that part.